components/dash-table/tests/selenium/test_filter.py

```diff
@@ -301,3 +301,40 @@ def test_filt003_sensitivity(test, filter_options, column_filter_options):
     else:
         assert target.cell(0, "c").get_text() == "abc"
         assert target.cell(1, "c").get_text() == "ABC"
+
+
+@pytest.mark.parametrize(
+    "column_placeholder_setting,table_placeholder_setting,expected_placeholder",
+    [
+        ("abc", None, "abc"),
+        (None, "def", "def"),
+        ("gah", "ijk", "gah"),
+        ("", None, ""),
+        (None, None, "filter data..."),
```

Comment:
Beautiful set of test cases! One additional case I'm curious about though: what if only `filter_options.case` is set at the column level, does the table-level `filter_options.placeholder` still make it through? ie does the column-level setting merge with or replace the table-level setting? Merging would be preferred.

Vice versa too, I suppose: does table-level `case` survive if only `placeholder` is supplied by the column?

Reply:
This is handled [here](https://github.com/plotly/dash/blob/2bd83a98322f95243359d846ec9e97a5b542a36e/components/dash-table/src/dash-table/dash/Sanitizer.ts#L85). Added the usecase to the tests for both placeholder_text and case.